src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1180,23 +1197,23 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
 
                                         $indent = 4
 
-                                        $errorCategoryMsg = & { Set-StrictMode -Version 1; $_.ErrorCategory_Message }
+                                        $errorCategoryMsg = $err.ErrorCategory_Message
 
                                         if ($null -ne $errorCategoryMsg)
                                         {
-                                            $indentString = '+ CategoryInfo          : ' + $_.ErrorCategory_Message
+                                            $indentString = '+ CategoryInfo          : ' + $err.ErrorCategory_Message
                                         }
                                         else
                                         {
-                                            $indentString = '+ CategoryInfo          : ' + $_.CategoryInfo
+                                            $indentString = '+ CategoryInfo          : ' + $err.CategoryInfo
                                         }
 
                                         $posmsg += ""`n"" + $indentString
 
-                                        $indentString = ""+ FullyQualifiedErrorId : "" + $_.FullyQualifiedErrorId
+                                        $indentString = ""+ FullyQualifiedErrorId : "" + $err.FullyQualifiedErrorId
```

Comment:
not sure about this one, it looks like it should be:
```suggestion
                                        $indentString += "" FullyQualifiedErrorId : "" + $err.FullyQualifiedErrorId
```

Reply:
This is the existing code that literally adds a `+` to the left side of the output.